osclib/stagingapi.py

```diff
@@ -587,10 +587,12 @@ def get_open_requests(self, query_extra=None):
         requests = []
 
         # xpath query, using the -m, -r, -s options
-        where = f"@by_group='{self.cstaging_group}' and @state='new'"
-        target = f"target[@project='{self.project}']"
+        if not match_filter:
+            target = f"target[@project='{self.project}']"
+            review_where = f"review[@by_group='{self.cstaging_group}' and @state='new']"
+            match_filter = f"state/@name='review' and {review_where} and {target}"
```

Comment:
should we expect the user to add the target project manually to the query when using the match_filter feature? Or can we populate it as well via `self.project`?  I ask since (IIRC) `self.project` is required anyway so the user might expect that (calling `staging -p <projectname> --match-filter "blabla"` might imply that the query should be run against the specified project)

Reply:
good catch, fixed.